remove some unneeded libraries at link time - Karl wants to keep them though in upstream
---
 texk/web2c/am/texmf.am |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/texk/web2c/am/texmf.am
+++ b/texk/web2c/am/texmf.am
@@ -161,7 +161,7 @@ else !WIN32
 # All the x_... and X_... variables will be empty if we aren't supporting X.
 # Follow the library order used in X11R6 itself:
 #   -lXaw -lXmu -lXt -lSM -lICE -lXext -lX11 (some may not be present).
-mf_x_libs = $(X_LIBS) $(x_tool_libs) $(X_PRE_LIBS) $(x_ext_lib) $(wlibs) $(X_EXTRA_LIBS)
+mf_x_libs = $(X_LIBS) $(x_tool_libs) $(wlibs) $(X_EXTRA_LIBS)
 endif !WIN32
 
 ## mf and mf-nowin compile mfextra.c with different CPPFLAGS.
